observability/observability.go

```diff
@@ -30,22 +30,22 @@ import (
 )
 
 var (
-	mReceiverReceivedSpans      = stats.Int64("otelcol/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
-	mReceiverDroppedSpans       = stats.Int64("otelcol/receiver/dropped_spans", "Counts the number of spans dropped by the receiver", "1")
-	mReceiverReceivedTimeSeries = stats.Int64("otelcol/receiver/received_timeseries", "Counts the number of timeseries received by the receiver", "1")
-	mReceiverDroppedTimeSeries  = stats.Int64("otelcol/receiver/dropped_timeseries", "Counts the number of timeseries dropped by the receiver", "1")
-
-	mExporterReceivedSpans      = stats.Int64("otelcol/exporter/received_spans", "Counts the number of spans received by the exporter", "1")
-	mExporterDroppedSpans       = stats.Int64("otelcol/exporter/dropped_spans", "Counts the number of spans received by the exporter", "1")
-	mExporterReceivedTimeSeries = stats.Int64("otelcol/exporter/received_timeseries", "Counts the number of timeseries received by the exporter", "1")
-	mExporterDroppedTimeSeries  = stats.Int64("otelcol/exporter/dropped_timeseries", "Counts the number of timeseries received by the exporter", "1")
+	mReceiverReceivedSpans      = stats.Int64("receiver/received_spans", "Counts the number of spans received by the receiver", "1")
+	mReceiverDroppedSpans       = stats.Int64("receiver/dropped_spans", "Counts the number of spans dropped by the receiver", "1")
+	mReceiverReceivedTimeSeries = stats.Int64("receiver/received_timeseries", "Counts the number of timeseries received by the receiver", "1")
+	mReceiverDroppedTimeSeries  = stats.Int64("receiver/dropped_timeseries", "Counts the number of timeseries dropped by the receiver", "1")
+
+	mExporterReceivedSpans      = stats.Int64("exporter/received_spans", "Counts the number of spans received by the exporter", "1")
+	mExporterDroppedSpans       = stats.Int64("exporter/dropped_spans", "Counts the number of spans received by the exporter", "1")
+	mExporterReceivedTimeSeries = stats.Int64("exporter/received_timeseries", "Counts the number of timeseries received by the exporter", "1")
+	mExporterDroppedTimeSeries  = stats.Int64("exporter/dropped_timeseries", "Counts the number of timeseries received by the exporter", "1")
```

Comment:
Also this is against the documentation in the opencensus which suggested to prefix metrics with the component name.

Reply:
By default, all metrics are prefixed with "otelcol" there is no need to repeat it here.